samples/client/petstore/php/SwaggerClient-php/phpunit.xml.dist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         stopOnFailure="false">
+    <testsuites>
+        <testsuite>
+            <directory>SwaggerClient-php/test/Api</directory>
```

Comment:
Hmmm...

If I run `vendor/bin/phpunit` no tests are executed. 

If I drop `SwaggerClient-php/` from the paths in this file the tests run fine.

A change to `phpunit.xml.mustache` could be to prefix `{{apiTestPath}}` etc. with `../` -- but maybe you have a bette alternative.

Overall this looks like a good improvement!

We should probably adjust `samples/client/petstore/php/SwaggerClient-php/pom.xml` as well:
```diff
diff --git a/samples/client/petstore/php/SwaggerClient-php/pom.xml b/samples/client/petstore/php/SwaggerClient-php/pom.xml
index faaeba8..5a0007c 100644
--- a/samples/client/petstore/php/SwaggerClient-php/pom.xml
+++ b/samples/client/petstore/php/SwaggerClient-php/pom.xml
@@ -47,9 +47,6 @@
                         </goals>
                         <configuration>
                             <executable>vendor/bin/phpunit</executable>
-                            <arguments>
-                                <argument>test</argument>
-                            </arguments>
                         </configuration>
                     </execution>
                 </executions>
```